dev-db/percona-xtrabackup/percona-xtrabackup-2.4.14.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
EAPI 7, please.

Reply:
Unfortunately, it fails to work with EAPI 7. I tried using emake instead of jinja, but then some weird notices about unsecure rpaths appeared. But most of all, the binaries were broken (missing crc libs).

I also checked mysql 5.7 / percona-server, they have the same base, but all use EAPI 6. Can we please leave this legacy percona-xtrabackup 2.4 branch at EAPI 6?

Reply:
Maybe you need `-DBUILD_SHARED_LIBS=OFF`.

Reply:
Yes, that does the trick, thanks!